docs/DATA_LAYER_IMPROVEMENT_PLAN.md

```
@@ -0,0 +1,645 @@
+    CREATE TABLE killmails_raw_2024_01 PARTITION OF killmails_raw
+      FOR VALUES FROM ('2024-01-01') TO ('2024-02-01');
+    
+    CREATE TABLE killmails_raw_2024_02 PARTITION OF killmails_raw
+      FOR VALUES FROM ('2024-02-01') TO ('2024-03-01');
+    
+    -- Continue for all needed months...
+    """
+    
+    # Migrate data in batches
+    execute """
+    INSERT INTO killmails_raw 
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)